oldScripts/pl_to_spreadsheet.py

```diff
@@ -0,0 +1,102 @@
+# Author: Naveen Nathan
+import time, requests, datetime
+
+import os.path
+import json
+import pandas as pd
+from dotenv import load_dotenv
+"""
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+"""
+
+# CS10 Su24 course id
+COURSE_INSTANCE_ID = 155812
+SERVER = "https://us.prairielearn.com/pl/api/v1"
+
+# This scope allows for write access.
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
+SPREADSHEET_ID = "16e6hWK4wWiqetuyJrDvBy4O9Wwp83NR1JptZU1dIIxI"
+GRADEBOOK_SPREADSHEET_ID = ""
+
+load_dotenv()
+PL_API_TOKEN = os.getenv("PL_API_TOKEN")
+
+"""
+Allows the user authenticate their google account, allowing the script to modify spreadsheets in their name.
+Borrowed from here: https://developers.google.com/sheets/api/quickstart/python
+"""
+
+
+def allow_user_to_authenticate_google_account():
```

Comment:
I don't see this used anywhere in the script. Is there a reason this is included?

Reply:
I was planning to use this to write to sheets in the future. Perhaps I should remove it from this PR, though.